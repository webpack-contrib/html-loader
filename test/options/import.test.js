/* eslint-disable */
import webpack from '../helpers/compiler';

describe('Options', () => {
  describe('import', () => {
    test('{Boolean}', async () => {
      const config = {
        loader: {
          test: /\.html$/,
          options: {},
        },
      };

      const stats = await webpack('options/import/fixture.js', config);
      const { source } = stats.toJson().modules[1];

      expect(source).toMatchSnapshot();
    });
  });
});
