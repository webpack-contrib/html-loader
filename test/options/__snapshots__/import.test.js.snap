// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options import {Boolean} 1`] = `
"// HTML Imports
import HTML__IMPORT__0 from './1.html';
import HTML__IMPORT__1 from '/2.html';
import HTML__IMPORT__2 from './1.html';
import HTML__IMPORT__3 from '/2.html';


// HTML Exports


// HTML
export default \`<!DOCTYPE html>
<html lang=\\"en\\">
<head>
  <meta charset=\\"utf-8\\">
  <title>HTML Loader</title>
</head>
<body>
  <!-- Ignore -->
  
  
  
  
  <!-- Transform -->
  \${HTML__IMPORT__0}
  \${HTML__IMPORT__1}

  \${HTML__IMPORT__2}
  \${HTML__IMPORT__3}
</body>
</html>
\`"
`;
